lock-api/src/main/java/com/palantir/lock/client/RemoteTimelockServiceAdapter.java

```diff
@@ -137,7 +137,11 @@ public long currentTimeMillis() {
 
     @Override
     public void close() {
-        transactionStarter.close();
-        commitTimestampGetter.close();
+        try {
+            transactionStarter.close();
+            commitTimestampGetter.close();
+        } catch (Exception e) {
```

Comment:
Looks like we don't actually close the `leaderTimeGetter`? Also log something if we actually throw here?

Reply:
Yes, this looks like another bug :(